lib/node_modules/@stdlib/blas/base/srotg/docs/types/test.ts

```diff
@@ -0,0 +1,112 @@
+/*
+* @license Apache-2.0
+*
+* Copyright (c) 2023 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+/// <reference types="@stdlib/types"/>
+
+import srotg = require( './index' );
+
+
+// TESTS //
+
+// The function returns a collection...
+{
+	srotg( 0.0, 2.0 ); // $ExpectType Collection
+}
+
+// The compiler throws an error if the function is provided an argument which is not a number...
+{
+	srotg( true, false ); // $ExpectError
+	srotg( false, true ); // $ExpectError
+	srotg( null, null ); // $ExpectError
+	srotg( undefined, undefined ); // $ExpectError
+	srotg( '5', '9' ); // $ExpectError
+	srotg( [], [] ); // $ExpectError
+	srotg( {}, {} ); // $ExpectError
+}
+
+// The compiler throws an error if the function is provided insufficient arguments...
+{
+	srotg(); // $ExpectError
+	srotg( 1.0, 2.0, 3.0 ); // $ExpectError
+}
+
+// Attached to the main export is an `assign` method which returns an array-like object containing numbers...
+{
+	const out = [ 0.0, 0.0, 0.0, 0.0 ];
+
+	srotg.assign( 0.0, 2.0, out, 1, 0 ); // $ExpectType Collection
+}
+
+// The compiler throws an error if the `assign` method is provided a first or second argument which is not a number...
+{
+	const out = [ 0.0, 0.0, 0.0, 0.0 ];
+
+	srotg.assign( true, true, out, 1, 0 ); // $ExpectError
```

Comment:
Same comment as above.

Reply:
If the above tests are updated, these should be updated, as well.

Reply:
@Pranavchiku This still needs to be updated.

Reply:
Updated.